client/blocks/comment-detail/style.scss

```diff
@@ -227,6 +227,18 @@
 	}
 }
 
+.comment-detail__site-icon-author-avatar {
+	position: relative;
+
+	.comment-detail__author-avatar-image {
+		border: 2px solid $white;
+		bottom: -7px;
+		position: absolute;
+		right: -7px;
+		width: 18px;
```

Comment:
Ugh, I never seem to remember about that. I always default to full-fledged alphabetical order. :/